mypyc/primitives/float_ops.py

```diff
@@ -166,3 +167,11 @@
     c_function_name="CPyFloat_IsNaN",
     error_kind=ERR_NEVER,
 )
+
+isinstance_float = function_op(
+    name="builtins.isinstance",
+    arg_types=[object_rprimitive],
+    return_type=bit_rprimitive,
+    c_function_name="PyFloat_Check",
```

Comment:
This should use `CPyFloat_Check`, since we always implicitly accept integers as compatible with `float`.

Reply:
discussed elsewhere and agreed to keep using `PyFloat_Check` here because otherwise expressions like `isinstance(1, float)` would return `True` and diverge from interpreted python.